### ISSUE TITLE
JAWS doesn't read aria-describedby label on a link if you use hot keys to navigate through types of elements

### ISSUE THREAD

**Issue by yujiahuo** [FreedomScientific/VFO-standards-support#719]
### Summary

> If you have an element like a link that has an aria-describedby description, that description won't be read if you navigate to it with hotkeys. If you tab over to it in a form, it will read the description.

### Expected result

> The description would be read consistently regardless of how you navigate to the element.

### Actual result

> The description is not read if you use hotkeys.

### Example

> ![MicrosoftTeams-image](https://github.com/FreedomScientific/VFO-standards-support/assets/14813356/efd4b67e-9349-4e73-a47c-f864d2f03a3a)

#### JAWS version and build number
2022.2211.7

#### Operating System and version
Windows 10

#### Browser and version:
Chrome 112.0.5615.138

**Comment by stevefaulkner** [comment]
Hi @yujiahuo can you provide a test case?

**Comment by stevefaulkner** [comment]
@yujiahuo  I created a [test case](https://codepen.io/stevef/pen/PoyXKbR) and can repoduce

@BrettLewisVispero JAWS 2023 announces the description when navigated to with the TAB key but not when arrowed to or U/V link navigation keys used. Is there a reason why this works this way?

**Comment by stevefaulkner** [comment]
**Note:** NVDA = Chrome behaviour:
does not announce description when using TAB to focus link, does not announce when using arrow keys. Does announce when using U/V link navigation keys.

**Comment by yujiahuo** [comment]
> Hi @yujiahuo can you provide a test case?

Oops, sorry! Thanks for making this.

**Comment by 36degrees** [comment]
We've just been looking into a very similar issue to try and understand the intended behaviour and found https://github.com/nvaccess/nvda/issues/9153#issuecomment-578381262 which suggests this might be working as designed (or at least was when the comment was posted back in 2020…).

**Comment by BrettLewisVispero** [comment]
Imported into ADO as bug 17971
